pom.xml

```diff
@@ -65,11 +65,13 @@
     <version.org.gwtbootstrap3>0.9.4</version.org.gwtbootstrap3>
     <version.org.wildfly.wildfly-dist>10.1.0.Final</version.org.wildfly.wildfly-dist>
     <version.org.wildfly-plugins.wildfly-maven-plugin>1.2.0.Final</version.org.wildfly-plugins.wildfly-maven-plugin>
+    <version.net.ltgt.gwt.maven.gwt-maven-plugin>1.0-rc-7</version.net.ltgt.gwt.maven.gwt-maven-plugin>
     <version.net.ltgt.gwt.maven.gwt-maven-plugin>1.0-rc-9</version.net.ltgt.gwt.maven.gwt-maven-plugin>
```

Comment:
duplicate version.net.ltgt.gwt.maven.gwt-maven-plugin

Reply:
Sorted alphabetically in b4220c6